packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/info.jsx

```diff
@@ -0,0 +1,125 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Popover, Money, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+const commission_tooltip_margin = 35;
+const stop_out_tooltip_margin = 135;
+
+const MultipliersInfo = ({
+    amount = 0,
+    commission = 0,
+    className,
+    currency,
+    multiplier = 0,
+    is_tooltip_relative,
+    should_show_tooltip,
+    should_show_percentage_tooltip,
+    stop_out = 0,
+}) => {
+    const commission_text = (
+        <Text
+            as='p'
+            line_height='s'
+            size='xxs'
+            className={classNames({
+                [`${className}-tooltip-text`]: className,
+            })}
+        >
+            <Localize
+                i18n_default_text='Commission <0/>'
+                components={[<Money key={0} amount={commission} currency={currency} show_currency />]}
+            />
+        </Text>
+    );
+
+    const stop_out_text = (
+        <Text
+            as='p'
+            line_height='s'
+            size='xxs'
+            className={classNames({
+                [`${className}-tooltip-text`]: className,
+            })}
+        >
+            <Localize
+                i18n_default_text='Stop out <0/>'
+                components={[<Money key={0} amount={stop_out} currency={currency} show_currency />]}
+            />
+        </Text>
+    );
+
+    const commission_tooltip = (
+        <Localize
+            i18n_default_text='<0>{{commission_percentage}}%</0> of (<1/> * {{multiplier}})'
+            values={{
+                commission_percentage: Number((commission * 100) / (multiplier * amount)).toFixed(4),
+                multiplier,
+            }}
+            components={[<span className='bold' key={0} />, <Money key={1} amount={amount} currency={currency} />]}
+        />
+    );
+
+    const stop_out_tooltip = (
+        <Localize
+            i18n_default_text='To ensure your loss does not exceed your stake, your contract will be closed automatically when your loss equals to <0/>.'
+            components={[<Money key={0} amount={stop_out} currency={currency} show_currency />]}
+        />
+    );
+
+    const stop_out_percentage_tooltip = (
+        <Localize
+            i18n_default_text='When your current loss equals or exceeds {{stop_out_percentage}}% of your stake, your contract will be closed at the nearest available asset price.'
+            values={{
+                stop_out_percentage: Math.abs(Number((stop_out * 100) / amount).toFixed(0)),
```

Comment:
```suggestion
                stop_out_percentage: Math.abs(Number((stop_out * 100) / amount).toFixed()),
```

Reply:
I'll use `Math.floor()` instead as it's the more performant option. Refer to [here](https://www.measurethat.net/Benchmarks/Show/5545/0/tofixed-vs-toprecision-vs-mathround-vs-mathfloorfast#latest_results_block)